csv-bulk-signature/src/main.test.ts

```diff
@@ -0,0 +1,54 @@
+import { getFileName, getFullNameFileName } from "./main";
+import { Contact } from "./types";
+
+const contactGenerator: () => Contact = () => ({
+  "Full Name*": "nothing",
+  Credentials: "nothing",
+  "Title*": "nothing",
+  "Office Phone*": "nothing",
+  "Mobile Phone": "nothing",
+  "Calendly Link": "nothing",
+  "Brand*": "ata-capital",
+});
+
+// test getFullNameFileName
+describe("getFullNameFileName", () => {
+  it("should return the correct file name", () => {
+    const contact: Contact = {
+      ...contactGenerator(),
+      "Full Name*": "John Doe",
+    };
+    const result = getFullNameFileName(contact);
+    expect(result.fullNameFileName).toBe("john_doe");
+  });
+
+  it("should return file name with a middle name", () => {
+    const contact: Contact = {
+      ...contactGenerator(),
+      "Full Name*": "John Michael Doe",
+    };
+    const result = getFullNameFileName(contact);
+    expect(result.fullNameFileName).toBe("john_michael_doe");
+  });
+});
```

Comment:
test for first+last and first+middle+last

Reply:
My pushback here is there isn't a case for middle name yet...but I thought it was worth exposing for how many people have middle names. 😄